src/test/java/org/springframework/data/jpa/repository/UserRepositoryTests.java

```diff
@@ -1175,6 +1175,48 @@ public void findByElementCollectionAttribute() {
 		assertThat(result).containsOnly(firstUser, secondUser);
 	}
 
+	@Test
```

Comment:
Please add references to the JIRA issue like the other test methods do. Also, please add new test methods *last*, not in the middle of others.